components/admin_console/jobs/table.jsx

```diff
@@ -0,0 +1,386 @@
+
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage, FormattedDate, FormattedTime, injectIntl, intlShape, defineMessages} from 'react-intl';
+
+import {createJob, cancelJob} from 'actions/job_actions.jsx';
+import {JobTypes, JobStatuses} from 'utils/constants.jsx';
+import ErrorStore from 'stores/error_store.jsx';
+
+const holders = defineMessages({
+    jobId: {
+        id: 'admin.jobTable.jobId',
+        defaultMessage: 'Job ID: '
+    },
+    lastActivityAt: {
+        id: 'admin.jobTable.lastActivityAt',
+        defaultMessage: 'Last Activity: '
+    },
+    runLengthSeconds: {
+        id: 'admin.jobTable.runLengthSeconds',
+        defaultMessage: ' seconds'
+    },
+    runLengthMinutes: {
+        id: 'admin.jobTable.runLengthMinutes',
+        defaultMessage: ' minutes'
+    },
+    cancelButton: {
+        id: 'admin.jobTable.cancelButton',
+        defaultMessage: 'Cancel'
+    }
+});
```

Comment:
Can you use `Utils.localizeMessage` instead? I think we're still trying to move away from using `injectIntl`

Reply:
I'll removed what I can, but I still need `injectIntl` because of `formatDate`.